dist/pinny.js

```diff
@@ -100,49 +100,54 @@
          */
         animation: {
             openComplete: function() {
-                var plugin = this;
-                this._disableExternalInputs();
-                this._focus();
+                setTimeout(function () {
```

Comment:
Ya, this example makes it look like RAF is more efficient http://jsfiddle.net/calpo/h7eee/.